crates/client-api/src/routes/database.rs

```diff
@@ -343,26 +329,18 @@ pub struct CatalogParams {
 }
 #[derive(Deserialize)]
 pub struct CatalogQueryParams {
-    expand: Option<bool>,
     #[serde(default)]
     module_def: bool,
 }
 pub async fn catalog<S>(
     State(worker_ctx): State<S>,
     Path(CatalogParams { name_or_address }): Path<CatalogParams>,
-    Query(CatalogQueryParams { expand, module_def }): Query<CatalogQueryParams>,
+    Query(CatalogQueryParams { module_def }): Query<CatalogQueryParams>,
     Extension(auth): Extension<SpacetimeAuth>,
 ) -> axum::response::Result<impl IntoResponse>
 where
     S: ControlStateDelegate + NodeDelegate,
 {
-    if module_def && expand.is_some() {
-        return Err((
-            StatusCode::BAD_REQUEST,
-            "expand and module_def cannot both be specified",
```

Comment:
@coolreader18 why was this constraint here? Does it invalidate the returned thing somehow? (something about the typespace?)

Reply:
Just because they do opposite things - expand makes the hacky representation more detailed, while module_def uses a non-hacky representation.